projects/shikshalokam/src/lib/core-module/constants/menu.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface MenuData {
```

Comment:
Where are you using this?? This should be in the interface folder.

Reply:
changed